pkg/controller/swiftproxy/swiftproxy_controller_test.go

```diff
@@ -242,15 +292,23 @@ func newExpectedDeployment(status apps.DeploymentStatus) *apps.Deployment {
 			OwnerReferences: []meta.OwnerReference{
 				{"contrail.juniper.net/v1alpha1", "SwiftProxy", "swiftproxy", "", &trueVal, &trueVal},
 			},
-			Labels: map[string]string{"SwiftProxy": "swiftproxy"},
+			Labels: map[string]string{"SwiftProxy": "swiftproxy", "contrail_manager": "SwiftProxy"},
```

Comment:
Do these labels persist?I think they are overwritten in controller.

Reply:
This expected deployment so basically this is what is expected to be generated from swiftproxy controller

Reply:
This expected deployment so basically this is what is expected to be generated from swiftproxy controller